specs/signature-specification.md

```diff
@@ -99,21 +57,21 @@ Besides the [image manifest property requirements][image-manifest-property-descr
 - **`subject`** (*descriptor*): A REQUIRED artifact descriptor referencing the signed manifest, including, but not limited to image manifest, image index, OCI artifact manifest.
 - **`annotations`** (*string-string map*): This REQUIRED property contains metadata for the artifact manifest.
   It is being used to store information about the signature.
-  Keys using the `io.cncf.notary` namespace are reserved for use in Notary and MUST NOT be used by other specifications.
+  Keys using the `io.cncf.notary` namespace are reserved for use in Notary Project and MUST NOT be used by other specifications.
   - **`io.cncf.notary.x509chain.thumbprint#S256`**: A REQUIRED annotation whose value contains the list of SHA-256 fingerprint of signing certificate and certificate chain (including root) used for signature generation. The annotation name contains the hash algorithm as a suffix (`#S256`) and can be extended to support other hashing algorithms in future.
     The list of fingerprints is present as a JSON array string, corresponding to ordered certificates in [*Certificate Chain* unsigned attribute](#unsigned-attributes) in the signature envelope.
 
 ### Signature Discovery
 
 The client should be able to discover all the signatures belonging to an artifact (such as image manifest) by using [OCI Distribution Referrers API][oci-distribution-referrers].
 OCI Distribution Referrers API returns a paginated list of all artifacts belonging to a target artifact (such as container images, SBoMs).
-The implementation can filter Notary signature artifacts by either using OCI Distribution Referrers API or using custom logic on the client.
-Each Notary signature artifact refers to a signature envelope blob.
+The implementation can filter Notary Project signature artifacts by either using OCI Distribution Referrers API or using custom logic on the client.
+Each Notary Project signature artifact refers to a signature envelope blob.
 
 ### Signature Filtering
 
-An OCI artifact can have multiple signatures, Notary v2 uses annotations of the signature artifact to filter relevant signatures based on the applicable trust policy.
-The Notary v2 signature artifact's `io.cncf.notary.x509chain.thumbprint#S256` annotations key MUST contain the list of SHA-256 fingerprints of certificate and certificate chain used for signing.
+An OCI artifact can have multiple signatures, Notary Project tooling uses annotations of the signature artifact to filter relevant signatures based on the applicable trust policy.
+The Notary Project signature artifact's `io.cncf.notary.x509chain.thumbprint#S256` annotations key MUST contain the list of SHA-256 fingerprints of certificate and certificate chain used for signing.
```

Comment:
Is it SHA-1 fingerprints or SHA-256 fingerprints for the certificates and certificate chains?

Reply:
It is SHA-256 fingerprints. SHA-1 is not secure.